### PR TITLE
Fix broken quarkusBuild task

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]
I can confirm that this PR fixes the issue

File: devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java [hunk 1]
```diff
@@ -132,6 +133,12 @@ public void buildQuarkus() {
 
         final AppArtifact appArtifact = extension().getAppArtifact();
         final AppModelResolver modelResolver = extension().resolveAppModel();
+        try {
+            // this needs to be done otherwise the app artifact doesn't get a proper path
+            modelResolver.resolveModel(appArtifact);
```
Comment (aloubyansky) [inline devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java]: Ah, ok, then modelResolver.resolve(appArtifact) should be enough. Resolving the model resolves all the deps + all the deployment deps, which is a waste in this case.

Comment (geoand) [reply devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java]: Let me give it a try.

Comment (geoand) [reply devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java]: I tried `modelResolver.resolve(appArtifact)` but the problem reappeared.

Comment (aloubyansky) [reply devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java]: yes, it's been awhile, the gradle's resolve doesn't support the resolve method. you can merge this pr. It'll cache the model and re-use it during the augmentation. So, not a waste.

**Review by aloubyansky — APPROVED** [review]

**Pull request merged**